python/GafferUI/FileSystemPathPlugValueWidget.py

```diff
@@ -65,7 +65,9 @@ def getToolTip( self ) :
 
 		extensions = self.__extensions()
 		if extensions :
-			result += "\n\nSupported file extensions : " + ", ".join( extensions )
+			if result :
+				result += "\n\n"
+			result += "## Supported file extensions\n\n " + "".join( "- {0}\n".format( e ) for e in extensions )
```

Comment:
This list is ignored in the node Reference because of [this line](https://github.com/GafferHQ/gaffer/blob/master/python/GafferUI/DocumentationAlgo.py#L218) in DocumentationAglo. Do we want to keep it as a sentence?

Reply:
I've pushed 87c8279, so that FileSystemPathPlugValueWidget now matches the node reference.